keps/prod-readiness/sig-windows/5100.yaml

```diff
@@ -4,3 +4,5 @@
 kep-number: 5100
 beta: 
   approver: "@soltysh"
+stable: 
+  approver: "@TBD"
```

Comment:
Please post on #prod-readiness on slack to get this no longer to say TBD.

Reply:
Just add me, I'll continue PRR-ing this :)

Reply:
Looks like this didn't get pushed. CI is still failing